src/routes/jira/server/jira-server-url-post.ts

```diff
@@ -3,34 +3,88 @@ import axios from "axios";
 import { GitHubServerApp } from "models/github-server-app";
 import { isValidUrl } from "utils/is-valid-url";
 
+interface MessageAndCode {
+	errorCode: string;
+	message: string;
+	statusCode: number;
+}
+
+interface GheServerUrlErrorResponses {
+	[key: string | number]: MessageAndCode;
+}
+
+interface GheServerUrlErrors {
+	codeOrStatus: GheServerUrlErrorResponses;
+}
```

Comment:
Not sure if you need this as there are not other keys at the top level, you can just remove `codeOrStatus` from the object